### ISSUE TITLE
Competition 

### ISSUE THREAD

**Issue by massatt212** [LukeShortCloud/winesapOS#816]
With other Competition why dont you focus on Desktop PC only, make an installer process and use Pure arch with your steam gamemode, cause everyone using fedora, Nobara doesnt feel like the SteamOS, Bazzite is very slow and Hangs a lot, chimera has a mouse problem atm and Holoiso doesnt install on every pc, giving an partition error, i think u may find more success in Desktop since everyone is focusing on handhelds when there are plenty people wanting to build their own console like PC.

**Comment by LukeShortCloud** [comment]
Hey @massatt212 , thanks for the suggestions! Desktop has been a big focus. It was not until winesapOS 3.4.0 that we finally added Game Mode. winesapOS 4.0.0 switched from SteamOS to Arch Linux. I started working on the installer but KDE Plasma 6 caused way too many issues that I have been working to fix lately. I will work on the installer again soon.

On the handheld side, I'm actually the Director of Linux Engineering over at Playtron. We're launching PlaytronOS in less than 2 months to allow people to play Steam, GOG, and Epic games all in one unified game console experience. Playtron is not focusing on desktop usage but winesapOS is. The great news is that my work on PlaytronOS is already benefing winesapOS!

Hopefully this provides some insight into the winesapOS roadmap. I'm open to any other suggestions you may have, too!

**Comment by wallentx** [comment]
Very cool to see that you get to do this for work stuff too!
Relating to the competition discussion, I've been tempted to make my own attempt at an arch-based distro focused on steam.... but with no desktop. Just booting to a Steam TUI. I like terminal tools, and I think it'd be nice to dump the overhead of plasma (or any other DE) stuff.
Any interest in WinesapOS having a terminal-only mode? 😁

**Comment by massatt212** [comment]
Everyone focus is on handhelds, forgetting about the desktop users lol, keep up the good work and stay healthy

**Comment by LukeShortCloud** [comment]
Thanks for all of the support from you two over the years! I'm very grateful. 🙏

winesapOS is not going anywhere. I still use it in desktop mode on a lot of my computers. It's really nice to have!

@wallentx You can try setting the environment variable `WINESAPOS_DE=none` before doing a build. I have not tested that use-case of having no desktop. I almost explored it once but there was no demand. Now that there is demand, I can help polish that if needed. 🙂

Here are some build hints:

https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-alpha.0/CONTRIBUTING.md#environment-variables-for-installation
https://github.com/LukeShortCloud/winesapOS/blob/4.1.0-alpha.0/CONTRIBUTING.md#automated-container-build

**Comment by LukeShortCloud** [comment]
I will go ahead and close this one out.

@massatt212 You can follow along with the winesapOS installer and converter discussion here: https://github.com/LukeShortCloud/winesapOS/issues/370

@wallentx If additional work is needed for headless winesapOS builds, feel free to open up a separate GitHub Issue. Hopefully `WINESAPOS_DE=none` will work for your build needs.
